rflx/pyrflx/typevalue.py

```diff
@@ -122,19 +136,18 @@ def _last(self) -> int:
         assert isinstance(last, Number)
         return last.value
 
-    def assign(self, value: int, check: bool = True) -> None:
+    def assign(self, value: int, length: int = 0) -> None:
         if (
-            self._type.constraints("__VALUE__", check).simplified(
+            self._type.constraints("__VALUE__", True).simplified(
```

Comment:
Is there any reason to remove the option to disable the type constraint check?

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/198/commits/1ab7b6311e8bc2f82f25c3c4b96d70a135337223.